.github/workflows/lint-test.yaml

```diff
@@ -147,7 +147,7 @@ jobs:
       - uses: taiki-e/install-action@v1.5.6
         with:
           tool: nextest
-      - run: cargo nextest run --profile ci --workspace --exclude smoke-test --exclude backup-cli --exclude testcases
+      - run: cargo nextest run --profile ci --workspace --exclude smoke-test --exclude backup-cli --exclude testcases --retries 3 --no-fail-fast
```

Comment:
i thought we already have retries to 3? https://github.com/aptos-labs/aptos-core/blob/6b4c39d61266e224c84ae1c02d2d967c6ca4ecf6/.config/nextest.toml#L13

it's in nextest.toml

Reply:
That is for smoke test, not unittest

Reply:
Also actually smoke test doesnt even use this, this is for smoke coverage

Smoke test also use retries flag on command line in workflow as well :P